error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -391,6 +391,25 @@ Flux<T> after(Flux<T> flux, T object) {
     }
   }
 
+  /** Prefer {@link Flux#empty()} over more contrived ways of creating an empty flux. */
```

Comment:
Usually we just say `more contrived alternatives`, so would say that would suffice here as well 😄.